.github/workflows/manubot.yaml

```diff
@@ -36,4 +36,10 @@ jobs:
           MANUBOT_SSH_PRIVATE_KEY: ${{ secrets.MANUBOT_SSH_PRIVATE_KEY }}
           BUILD_WEB_URL: https://github.com/${{ github.repository }}/commit/${{ github.sha }}/checks
         shell: bash --login {0}        
-        run: bash ci/deploy.sh
+        run: |
+          if [ "$MANUBOT_SSH_PRIVATE_KEY" != "" ]; then
```

Comment:
This would be better in an if expression for the step, but I couldn't figure out how to do that.
https://github.community/t5/GitHub-Actions/If-expression-with-context-variable/m-p/34560/highlight/true#M1959